index.js

```diff
@@ -31,6 +31,7 @@ var Parser = function (opts) {
   this.strict = opts.strict || null
   this.mapHeaders = opts.mapHeaders || identity
   this.mapValues = opts.mapValues || identity
+  this.skipUntil = opts.skipUntil || null
```

Comment:
let's use `skipLines` here instead, as that's more descriptive of the feature

Reply:
all set!